docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -0,0 +1,180 @@
+====
+
+[discrete]
+=== Step 2: Configure {elastic-endpoint}
+
+Set the `advanced.artifacts.global.base_url` advanced setting for each <<configure-endpoint-integration-policy,{elastic-defend} integration policy>> that needs to use the mirror. Note that there's a separate setting for each operating system:
+
+* `linux.advanced.artifacts.global.base_url`
+* `mac.advanced.artifacts.global.base_url`
+* `windows.advanced.artifacts.global.base_url`
+
+[role="screenshot"]
+image::images/offline-adv-settings.png[Integration policy advanced settings]
+
+[discrete]
+=== Step 3: Manually copy artifact updates
+
+Download the most recent artifact files from the Elastic global artifact server, then copy those files to the server instance you created in step 1.
+
+Below is an example script that downloads all the global artifact updates. There are different artifact files for each version of {elastic-endpoint}. Change the value of the `ENDPOINT_VERSION` variable in the example script to match the deployed version of {elastic-endpoint}.
+
+[source,sh,subs="attributes"]
+----
+export ENDPOINT_VERSION={version} && wget -P downloads/endpoint/manifest https://artifacts.security.elastic.co/downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip && zcat -q downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip | jq -r --arg root "https://artifacts.security.elastic.co" '.artifacts | to_entries[] | $root + .value.relative_url' | xargs wget -P downloads/endpoint -c
+----
+
+This command will download files and directory structure that should be directly copied to the file server.
+
+Elastic releases updates on a continuous basis as detection engines are improved. We recommend updating air-gapped environments monthly to stay current with artifact updates.
+
+[discrete]
+[[validate-artifact-server]]
+== Validate your self-hosted artifact server
+
+Each new global artifact update release increments a version identifier that you can check to ensure that {elastic-endpoint} has received and installed the latest version.
+
+To confirm the current latest version of the artifacts for a given {elastic-endpoint} version, check the published version. This example script checks the version:
+
+[source,sh,subs="attributes"]
+----
+curl -s https://artifacts.security.elastic.co/downloads/endpoint/manifest/artifacts-{version}.zip | zcat -q | jq -r .manifest_version
+----
+
+Replace `https://artifacts.security.elastic.co` in the command above with your local mirror server to validate that the artifacts are served correctly.
+
+After updating {elastic-endpoint} configuration to read from the mirror server, use {kib}'s Discover view to search for `endpoint.policy` response documents, then check the installed version (`Endpoint.policy.applied.artifacts.global.version`) and compare with the output from the command above:
```

Comment:
Maybe this is obvious from the screenshot, but we may want to mention these documents are found in the `metrics-*` data view (as opposed to logs)

Reply:
```suggestion
After updating {elastic-endpoint} configuration to read from the mirror server, use {kib}'s Discover view to search the `metrics-*` data view for `endpoint.policy` response documents, then check the installed version (`Endpoint.policy.applied.artifacts.global.version`) and compare with the output from the command above:
```